guides/v2.2/install-gde/basics/basics_software.md

```diff
@@ -18,7 +18,7 @@ functional_areas:
 
 The full list of software is listed in [System Requirements]({{ page.baseurl }}/install-gde/system-requirements.html) but here are the essentials:
 
-* Web server for providing access to web pages - Currently, we've documented [Apache](http://en.wikipedia.org/wiki/Apache_HTTP_Server){:target="_blank"} and plan to document other web servers in the near future.
+* Web server for providing access to web pages - Currently, we've documented [Apache](http://en.wikipedia.org/wiki/Apache_HTTP_Server){:target="_blank"} and [NGINX](https://de.wikipedia.org/wiki/Nginx){:target="_blank"}. We also plan to document other web servers in the near future.
```

Comment:
Please, provide a link to the English Wiki version

Reply:
fixed: https://github.com/magento/devdocs/pull/5597/commits/64aab397a17546674594179cf222e0c115c9f027